stdlib/time.pyi

```diff
@@ -63,14 +63,14 @@ class struct_time(structseq[Any | int], _TimeTuple):
     @property
     def tm_gmtoff(self) -> int: ...
 
-def asctime(t: _TimeTuple | struct_time = ...) -> str: ...
-def ctime(secs: float | None = ...) -> str: ...
-def gmtime(secs: float | None = ...) -> struct_time: ...
-def localtime(secs: float | None = ...) -> struct_time: ...
-def mktime(t: _TimeTuple | struct_time) -> float: ...
-def sleep(secs: float) -> None: ...
-def strftime(format: str, t: _TimeTuple | struct_time = ...) -> str: ...
-def strptime(string: str, format: str = ...) -> struct_time: ...
+def asctime(time_tuple: _TimeTuple | struct_time = ..., /) -> str: ...
+def ctime(seconds: float | None = None, /) -> str: ...
+def gmtime(seconds: float | None = None, /) -> struct_time: ...
+def localtime(seconds: float | None = None, /) -> struct_time: ...
```

Comment:
The `None` default comes from [here](https://github.com/python/cpython/blob/5e1916ba1bf521d6ff9d2c553c057f3ef7008977/Modules/timemodule.c#L513)